extension/content_script.js

```diff
@@ -132,9 +131,6 @@ function calculateTrackerLife() {
 	const speed_bias_ms = 500; // Half of a second; is this too long?
 
 	let sentenceId = tracker.getSentenceId();
-	let sentencePtr = doc.getSentence(sentenceId);
-
-	let containerId = sentencePtr.containerId;
```

Comment:
I believe these are not used in this function

Reply:
Nice, thanks for catching this!